### PR TITLE
Updating to support NServiceBus 6.0.0-RC3

### PR DESCRIPTION
**Pull request by andreasohlund**
Other changes:
- Removed ripple
- Removed the settings for using envelope or not for multi messages since v6 no longer allows multi messages to be sent (can still be received)

### CODE REVIEW

File: ripple.config [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<ripple xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<ripple xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```
Comment (fhalim) [inline ripple.config]: Can ripple related files be deleted if the build doesn't use it anymore?

**Review by fhalim — CHANGES_REQUESTED** [review]
This looks great. Could you remove the ripple related files?

File: ripple.config (hunk 1 above)
Comment (andreasohlund) [reply ripple.config]: Duh, good catch. Will do

Comment (andreasohlund) [reply ripple.config]: Ripple is now removed

**Comment by andreasohlund** [comment]
@fhalim should be good to go

Side note: Is this something you want to maintain going forwards or would you like to transfer this into https://github.com/ParticularLabs so that we can try to get more users involved?

**Comment by fhalim** [comment]
I'd love to transfer ownership. Let me know how we can go about doing that. Incidentally, this PR looks great.

**Comment by andreasohlund** [comment]
Merge this then we can sort out the transfer?

**Comment by fhalim** [comment]
Done. Thanks for the PR. Pushed out a release on nuget.org.

**Pull request merged**